src/tui/rendering.rs

```diff
@@ -127,10 +138,10 @@ impl Tui {
         self.process_table.selected()
     }
 
-    pub fn update_number_of_items(&mut self, number_of_items: usize) {
-        self.number_of_items = number_of_items;
-        self.process_table_scroll = self
-            .process_table_scroll
+    pub fn update_process_table_number_of_items(&mut self, number_of_items: usize) {
```

Comment:
Renamed this method according to the variable name.

Reply:
Ok